src/tr1/config.h

```diff
@@ -121,6 +121,7 @@ typedef struct {
         int32_t turbo_speed;
         bool pretty_pixels;
         bool enable_reflections;
+        bool enable_ui;
```

Comment:
I think the `ui` struct would be a more suitable place to put this setting. WDYT?

Reply:
Absolutely. Done 👍